### ISSUE TITLE
Memory leak when running backup

### ISSUE THREAD

**Issue by zhfnjust** [mceSystems/libijs#5]
Thanks for the repo.

i use the lib to backup my phone data. i work well but  a Memory leak.
i can see the using Memory increase in the window task and the process crash finally when run backup action.
i have no idea to fix the bug.

**Comment by gara001001** [comment]
yes, I have the same problem when the ios device data is very large , and it will cost all memery until my computer slows down.
